fitwsarah_frontend/src/views/authentication/Signup.js

```diff
@@ -71,13 +89,18 @@ function AddMemberProfile() {
 
     useEffect(() => {
         if (accessToken) {
+            const storedUserData = localStorage.getItem("userData");
```

Comment:
what does localStorage is used for?

Reply:
It used for persistence to save user data through the web browser, it sort of acts as a session.